examples/v2/filter_subscriber.nim

```diff
@@ -22,7 +23,7 @@ proc unsubscribe(wfc: WakuFilterClient,
                  filterPubsubTopic: PubsubTopic,
                  filterContentTopic: ContentTopic) {.async.} =
   notice "unsubscribing from filter"
-  let unsubscribeRes = await wfc.unsubscribe(filterPeer, filterPubsubTopic, @[filterContentTopic])
+  let unsubscribeRes = await wfc.unsubscribe(filterPeer, some(filterPubsubTopic), @[filterContentTopic])
```

Comment:
imho pubsub topic should be derived directly from content topic.
* simpler
* less error prone (what if the provided pubsub topic doesnt match the one derived from content topic? it introduces a new not neccesary error path)

in other words

unsubscribe(filterPeer, @[filterContentTopic])